api/pkg/cli/cmd/info/info.go

```diff
@@ -139,8 +139,68 @@ func (opts *options) run() error {
 	}
 
 	hubClient := opts.cli.Hub()
+
+	// Get all Catalogs
+	catalog, err := hubClient.GetCatalogsList()
```

Comment:
this can be done when user has not passed catalog
```
if opt.from == "" {
  get all catalogs
}
```

Reply:
I think in order to keep default catalog as `tekton` as mentioned [here](https://github.com/tektoncd/hub/pull/265#discussion_r637683325) we'll have to make an api call to get all catalogs

Reply:
you can keep the `from` flag empty but if user passes a catalog through the flag then you don't need to `get all catalogs`.
use that catalog to fetch the resource, whether it exist or not.

Reply:
Yes make sense to get all the catalog if we keep the flag empty

Reply:
Updated the PR!